### PR TITLE
add t to README.md

### PR DESCRIPTION
**Pull request by mkonxd**
PR created in collaboration with [Lucas Høyberg Puvis de Chavannes](https://github.com/lucaschavannes)

Very cool tool! Excited to see it being used on some of the big LLMs out there :)

### CODE REVIEW

**Comment by mkonxd** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mkonxd** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # garak, an LLM vulnerability scanner
 
 `garak` checks if an LLM will fail in an way we don't necessarily want. `garak` probes for hallucination, data leakage,
-promp injection, misinformation, toxicity generation, jailbreaks, and many other weaknesses. It's a free tool. We love developing
+prompt injection, misinformation, toxicity generation, jailbreaks, and many other weaknesses. It's a free tool. We love developing
 it and are always interesting in adding functionality to support applications. Join our [Discord](https://discord.gg/xH3rs3ZH4B)!
```
Comment (lucaschavannes) [inline README.md:5]: Wouldn't it be _interesting_ -> _interested?_

Comment (mkonxd) [reply README.md:5]: you're right, i'll fix

**Comment by leondz** [comment]
Wow you guys are getting PRs merged on some sick repos these days, gj!! Great   to see you here!

**Pull request merged**